exporter/opentelemetry-exporter-datadog/CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+## Unreleased
+
```

Comment:
I think we are using one [CHANGELOG](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/master/CHANGELOG.md) file now.

Reply:
Actually, this change does not seem to be user-facing, it is actually a bug fix. I have removed this CHANGELOG file.